scripts/test-cli.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# test-cli.sh
+#
+# SUMMARY
+#
+#   Run cli tests only.
+
+cargo test --test cli --features cli-tests -- --test-threads 4
```

Comment:
I don't think this file is required? The makefile does this already.

Reply:
True, force of habit.